ANNOTATION.md

```diff
@@ -33,109 +33,109 @@
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Try'\]/\*\[@internalRole\]\[@internalRole='orelse'\] | IfElse |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='TryExcept'\] | TryCatch |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='ExceptHandler'\] | TryCatch |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='TryFinally'\] | TryFinally |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Raise'\] | Throw |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='With'\] | BlockScope |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Return'\] | Return |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Break'\] | Break |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Continue'\] | Continue |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\] | If |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='TryExcept'\] | TryCatch, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='ExceptHandler'\] | TryCatch, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='TryFinally'\] | TryFinally, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Raise'\] | Throw, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='With'\] | BlockScope, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Return'\] | Return, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Break'\] | Break, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Continue'\] | Continue, Statement |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\] | If, Statement |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='If\.body'\] | IfBody |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@internalRole\]\[@internalRole='test'\] | IfCondition |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='If\.orelse'\] | IfElse |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\] | BinaryExpression |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\] | BinaryExpression, Expression |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@InternalType='Compare\.ops'\] | BinaryExpressionOp |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@InternalType='Compare\.comparators'\] | BinaryExpressionRight |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@internalRole\]\[@internalRole='left'\] | BinaryExpressionLeft |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression, Statement |
```

Comment:
I tough so, but it seems it's only an expression (https://help.semmle.com/wiki/display/QL/Tutorial%3A+Statements+and+Expressions).